### ISSUE TITLE
Can't add NSBgeek indexer

### ISSUE THREAD

**Issue by xdroop** [Radarr/Radarr#1963]
**Description:**

New to Radarr. Just set up Sonarr.

I am trying to add my NZBgeek account as an indexer so that I have something to search.

When I hit "test", I get the "Unable to connect to indexer" message.

Logs look like this at a high level:

```
17-8-18 21:21:42.4|Debug|NewznabCapabilitiesProvider|Failed to get newznab api capabilities from https://api.nzbgeek.info
17-8-18 21:21:42.4|Warn|Newznab|Unable to connect to indexer
17-8-18 21:21:42.6|Debug|NewznabCapabilitiesProvider|Failed to get newznab api capabilities from https://api.nzbgeek.info
17-8-18 21:21:42.6|Warn|Newznab|Unable to connect to indexer: Error: SecureChannelFailure (Object reference not set to an instance of an object)
17-8-18 21:21:42.6|Warn|NzbDroneErrorPipeline|Invalid request Validation failed:
17-8-18 21:21:42.6|Debug|Api|[POST] /api/indexer/test: 400.BadRequest (473 ms)
```

Checked the Search and there doesn't appear to be any current discussion about NZBgeek.

**Radarr Version:**  Radarr.develop.0.2.0.778.linux.tar.gz

**Logs:** [github-2017-08-18.txt](https://github.com/Radarr/Radarr/files/1236001/github-2017-08-18.txt)

I'm probably doing something wrong again, I just can't see what it is.

**Comment by galli-leo** [comment]
What version of mono are you using? Seems like the https is failing.

**Comment by xdroop** [comment]
```
# mono --version
Mono JIT compiler version 5.0.1.1 (2017-02/5077205 Wed May 24 13:16:08 UTC 2017)
```

This is from the Yum repository http://download.mono-project.com/repo/centos/.

I have Sonarr running on the same computer, which works fine with the same information.

**Comment by galli-leo** [comment]
Can you try using the fix suggested here #1528 or downgrading mono to 4.8?

**Comment by xdroop** [comment]
The fix suggested worked so far.
